### PR TITLE
Reduce docker layers further

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

**Comment by hennevogel** [comment]
Hm, why? You make the number of layers lower and the diff between them larger. What's the point?

File: Dockerfile.frontend-base [hunk 1]
```diff
@@ -5,9 +5,7 @@ RUN /root/bin/docker-bootstrap.sh frontend
 # Install other requirements
 RUN npm install -g jshint
 
-ADD src/api/Gemfile /obs/src/api/Gemfile
-ADD src/api/Gemfile.lock /obs/src/api/Gemfile.lock
-RUN chown -R frontend /obs/src/api
+ADD --chown=frontend:users src/api/Gemfile* /obs/src/api/
```
Comment (eduardoj) [inline Dockerfile.frontend-base]: Nitpick. Is `:users` needed? I would leave this line as:

`ADD --chown=frontend src/api/Gemfile* /obs/src/api/`

File: Dockerfile.old-test-suite [hunk 2]
```diff
@@ -4,10 +4,8 @@ RUN /root/bin/docker-bootstrap.sh memcached
 RUN /root/bin/docker-bootstrap.sh backend
 RUN /root/bin/docker-bootstrap.sh frontend
 
-ADD src/api/Gemfile /obs/src/api/Gemfile
-ADD src/api/Gemfile.lock /obs/src/api/Gemfile.lock
+ADD --chown=frontend:users src/api/Gemfile* /obs/src/api/
```
Comment (eduardoj) [inline Dockerfile.old-test-suite]: Nitpick. Same as previous comment.

Is `:users` needed? I would leave this line as:

`ADD --chown=frontend src/api/Gemfile* /obs/src/api/`

File: Dockerfile [hunk 3]
```diff
@@ -9,9 +9,7 @@ ARG CONTAINER_USERID
 # Configure our user
 RUN usermod -u $CONTAINER_USERID frontend
 
-ADD src/api/Gemfile /obs/src/api/Gemfile
-ADD src/api/Gemfile.lock /obs/src/api/Gemfile.lock
-RUN chown -R frontend /obs/src/api
+ADD src/api/Gemfile* /obs/src/api/
```
Comment (eduardoj) [inline Dockerfile]: Nitpick. I would add here the `chown`, and leave it as:

`ADD --chown=frontend src/api/Gemfile* /obs/src/api/`

**Pull request closed without merging**